Some additions and corrections regarding the Japanese translation

hope366 opened the pull request:
I rejected the PR I created a while ago and created a new one because it contained inappropriate content. 
>1477 AnalysisFrame.checkGraph
1478 AnalysisFrame.checkList
1479 AnalysisFrame.checkShowNext
1480 AnalysisFrame.checkUseMouseMove

These four items used `<html> <span style =" font-family: Yu Gothic UI; ">` to use the fonts in the Yu Gothic UI. 
If there is a smarter way, I'd like to remove it later. 

By the way, the red frame in the screenshot below is written in Chinese. Is this a feature to bring the window for contributions to the front?  Is it possible to make this multilingual? 
![無題](https://user-images.githubusercontent.com/63999713/152681854-55482fd3-71e9-4a03-b485-5a1d31eeb32a.png)

yzyray commented:
I missed to translate `总在最前`,it means always on top.
I have fixed the font about `AnalysisFrame.checkGraph` and similar items,it should be Yu Gothic UI by default.

hope366 commented:
I removed the html tag, but it is displayed in the beautiful font of Yu Gothic UI.
In addition to the four items in the candidate list with the html tag removed, the part that was not displayed in Yu Gothic UI has become a beautiful font in Yu Gothic UI 😄
Thank you for your support. 
Thank you for making the "front view" of the contribution panel multilingual. I added it to a Japanese file.
